api-java-mixin.raml

```diff
@@ -568,6 +582,11 @@ types:
       public static com.commercetools.api.models.common.ReferenceTypeId referenceTypeId() {
           return com.commercetools.api.models.common.ReferenceTypeId.TYPE;
       }
+
+      @Nullable
+      default FieldDefinition getFieldDefinitionByName(final String name) {
+          return getFieldDefinitions().stream().filter(def -> def.getName().equals(name)).findFirst().orElse(null);
+      }
```

Comment:
Should be put to the TypeMixin

Reply:
I tried but it didn't work for me

Reply:
you have to define the getFieldDefinitions() stub in the TypeMixin interface. Then it should work